packages/react-components/src/molecules/Modal.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import { Card, Overlay } from '../atoms';
+
+import {
+  vminLinearCalcClamped,
+  mobileScreen,
+  largeDesktopScreen,
+  tabletScreen,
+} from '../pixels';
+
+const overlayContainerStyles = css({
+  position: 'fixed',
+  width: '100%',
+  height: '100%',
+  top: 0,
+  left: 0,
+});
+
+const modalContainerStyles = css({
+  top: 0,
+  left: 0,
+  position: 'absolute',
```

Comment:
Won't transparency be an issue then?

Reply:
the overlay container would have 100% opacity, the overlay less than 100%, and the modal 100% but it would only cover part of the area if layed out correctly via margin or its containing grid/flex